packages/utils/src/schema/getDefaultFormState.ts

```diff
@@ -93,28 +100,45 @@ function maybeAddDefaultToObject<T = any>(
   }
 }
 
+interface computeDefaultsProps<T = any, S extends StrictRJSFSchema = RJSFSchema> {
```

Comment:
For consistency, all interfaces and types use PascalCase.

```suggestion
interface ComputeDefaultsProps<T = any, S extends StrictRJSFSchema = RJSFSchema> {
```